modules/cache/variables.tf

```diff
@@ -54,7 +54,7 @@ variable "cache_lifecycle_clear" {
 variable "cache_lifecycle_prefix" {
   description = "Object key prefix identifying one or more objects to which the clean up rule applies."
   type        = string
-  default     = "runner/"
+  default     = "project/"
```

Comment:
The root directory is runner in the bucket, for that reason I had set the default to runner. What is the issue here?

Reply:
Checking my current deployment, to see if it is not cleared. Will update in a few days.

Reply:
Strange. The root directory is `project/` in my case and I am using the default values only.

The lifecycle rule says `runner/` thus no clean up is done here.

I will also doublecheck my code.

Reply:
Just doubled checked, deployed the default example (examples dir) and cache files are stored in runners/<id>/project. So the root dir in my case is runner

Reply:
Strange. I will revert this change.